app/controllers/users_controller.rb

```diff
@@ -62,9 +62,36 @@ def filters
     end
   end
 
+  def make_owner_page
+    add_breadcrumb t('users.make_owner'), :make_owner_page_users_path
+    admins = @users.where(role: User::ROLES[1])
+    staff = @users.where(role: User::ROLES[0])
```

Comment:
how many roles are there in the application?

Reply:
3. It was just to exclude owner in the listing

Reply:
then use where.not and exclude owner in one query rather than 2.